compiler/passes/convert-typed-uast.cpp

```diff
@@ -5511,7 +5511,30 @@ Expr* TConverter::convertFieldAccessOrNull(const AstNode* node, RV& rv) {
   // Otherwise we'll do our best to generate a "plain-old" field access.
   types::QualifiedType qtField;
   auto [recvAst, fieldName] = accessExpressionDetails(node);
-  auto get = codegenGetField(recvAst, fieldName.c_str(), rv, &qtField);
+
+  types::QualifiedType qtRecv;
+  Expr* recv = recvAst ? convertExpr(recvAst, rv, &qtRecv) : nullptr;
+
+  if (recvAst && qtRecv.type()->isClassType() &&
```

Comment:
Note that a design choice has been to try treat `_owned` and `owned` as interchangeable. Thus, there might be cases where something is not a class type, but ought to be treated like one. See: 

https://github.com/chapel-lang/chapel/blob/4bbefc92451a4ea4f344775fb6172e077392196f/frontend/include/chpl/resolution/can-pass.h#L205-L207

Reply:
I'll note this as a TODO, pending a case where I encounter it in the wild.